### ISSUE TITLE
Proxy settings are ignored

### ISSUE THREAD

**Issue by rkitron** [PaloAltoNetworks/terraform-provider-prismacloud#6]
## Describe the bug

My network has corporate proxy setup via environment variables.

## Expected behavior

Outgoing connection to Prisma API use the proxy

## Current behavior

Timeout exception

## Possible solution

Within prima-cloud-go library:
```
	if c.Transport == nil {
		c.Transport = &http.Transport{
			TLSClientConfig: &tls.Config{
				InsecureSkipVerify: c.SkipSslCertVerification,
			},
			Proxy: http.ProxyFromEnvironment,
		}
	}
```
Thanks,
Benoit

**Comment by shannonrdunn** [comment]
Also experiencing this issue

**Comment by ebeuerle** [comment]
Fixed in v1.0.4 of the provider.
